plugin-offline/src/main/java/com/mapbox/mapboxsdk/plugins/offline/MapboxOffline.java

```diff
@@ -0,0 +1,39 @@
+package com.mapbox.mapboxsdk.plugins.offline;
+
+import android.app.Activity;
+import android.content.Intent;
+
+import com.mapbox.mapboxsdk.offline.OfflineTilePyramidRegionDefinition;
+import com.mapbox.mapboxsdk.plugins.offline.ui.OfflineActivity;
+
+import static com.mapbox.mapboxsdk.plugins.offline.Constants.RETURNING_DEFINITION;
+import static com.mapbox.mapboxsdk.plugins.offline.Constants.RETURNING_REGION_NAME;
+
+public class MapboxOffline {
```

Comment:
the name implicates a general entry point to offline/plugin but the class is a utility for all things related to intents. MapboxOfflineIntent would be a better name?

Reply:
Renamed to `OfflineRegionSelector` to keep consistency between places and this plugin.